pkg/notifier/email.go

```diff
@@ -0,0 +1,53 @@
+package notifier
+
+import (
+	"fmt"
+	"github.com/go-mail/mail"
```

Comment:
If you will have a look at any major golang repo like [this](https://github.com/kubernetes/kubernetes/blob/master/pkg/client/conditions/conditions.go#L19) or even golang [docs](https://golang.org/doc/code.html#Library) you will find out that there is some "standard" in how to format imports. It's 1 empty line after golang-specific imports, then project imports. I almost sure it's the standard in [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) as well